src/xdocs/google_style.xml

```diff
@@ -698,8 +698,8 @@
                   </a>
                 </td>
                 <td>
-                  <a href="styleguides/google-java-style-20180523/javaguide.html#s4.1.3-braces-empty-blocks">
-                    4.1.3 Empty blocks: may be concise</a>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s4.1.3-braces-empty-blocks">
+                    4.1.3 Empty Blocks</a>
```

Comment:
Had to modify the section name a bit so there'd no problem in validating the chapter wise testing in XdocPagesTest

Reply:
Nvm, added support properly process the section name.

https://github.com/checkstyle/checkstyle/blob/c2d5e67b454c451fd4b1f37bff5193ed62bf6ae0/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L1971

https://github.com/checkstyle/checkstyle/blob/c2d5e67b454c451fd4b1f37bff5193ed62bf6ae0/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L1999-L2023

New name for test class:

https://github.com/checkstyle/checkstyle/blob/c2d5e67b454c451fd4b1f37bff5193ed62bf6ae0/src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java#L26